Modules/Filtering/ImageStatistics/include/itkAdaptiveHistogramEqualizationImageFilter.h

```diff
@@ -47,7 +47,8 @@ namespace itk
  * (beta=1, with alpha=1).
  *
  * The parameter window controls the size of the region over which
- * local statistics are calculated.
+ * local statistics are calculated. The size of the window is controlled
```

Comment:
controlled **by**

Reply:
Good catch -- fixed!

Reply:
This is why I can't edit for grammar for anyone's paper :-)